frontend/src/Sidebar.res

```diff
@@ -11,30 +11,51 @@ let pullToString = ((pull_number, branch)) =>
   | None => "#" ++ Belt.Int.toString(pull_number)
   }
 
-let pullsMenu = (~pulls) => {
-  <Column>
-    <Text color=Sx.gray700 weight=#bold uppercase=true size=#md> {Rx.text("Pull Requests")} </Text>
-    {pulls
-    ->Belt.Array.mapWithIndex((i, pull) =>
-      <Link key={string_of_int(i)} href={linkForPull(pull)} text={pullToString(pull)} />
-    )
-    ->Rx.array}
-  </Column>
+module PullsMenu = {
+  @react.component
```

Comment:
I'm never sure when something needs to be an actual React component or just a function producing an element.

My convention so far has been to create a new component if:
1. The view needs to manage its own state.
2. The view is expensive to render.

But I have no idea if this has any meaningful implications. Do you have any thoughts on this?

Reply:
I think that makes sense. As long as you don't use `state`, `useEffect`, `memo` et al, both should be fine (I'm not sure it does matter much in this case)